src/App.jsx

```diff
@@ -20,6 +21,7 @@ const App = () => {
       <Route path="/contact" element={<Contact />} />
       <Route path="/favourites" element={<Favourites />} />
       <Route path="/about" element={<About />} />
+      <Route path="/filter" element={<Filter />} />
```

Comment:
I don't think we need a whole page for the filter?

Reply:
removed